.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
   artifacts:
     docker:
-      - image: circleci/python:3.7.6-stretch-node-browsers
+      - image: circleci/python:3.7.9-stretch-node-browsers
```

Comment:
Bump to latest patch version for each Python version, side-effect is that this updates the version of Chrome

Reply:
Two thoughts:
1. should we switch to circleci's "next-gen" images? https://circleci.com/developer/images/image/cimg/python
2. Maybe a bigger task, but should we be switching from py3.7 to py3.9?

Reply:
@alexcjohnson I can certainly look into those. If possible I'd like to do these in a follow up as this work was initiated by an issue @harryturr encountered as part of his current work assignment.